Assets/PlayerMovement.cs

```diff
@@ -6,7 +6,8 @@ public class PlayerMovement : MonoBehaviour
 {
     public float Speed = 1f;
     public float JumpVelocity = 10;
-    
+    public static int Score = 0; // The total number of collected coins
```

Comment:
1) `Score` does not conceptually belong in `PlayerMovement`. It should be in a separate class
2) Usage of static fields is not advised. First, its global state. Second, static fields do not necessarily get reset when restarting a game. In Editor, there is a possibility to run the game without domain reload (try this feature, it's extremely convenient; it makes your game start almost immediately).  Project Settings/Editor 
![image](https://user-images.githubusercontent.com/32465621/154425534-5a0b441e-eb2f-4a24-a86b-7110349e2d8e.png)
3) Such public fields violate encapsulation (especially when they are static). It would be better to have a property with public getter and private setter and a method that adds a coin.